subliminal/providers/podnapisi.py

```diff
@@ -41,7 +46,9 @@ def compute_matches(self, video):
         # episode
         if isinstance(video, Episode):
             # series
-            if video.series and self.series.lower() == video.series.lower():
+            if video.series and \
+               IGNORED_CHARACTERS_RE.sub('', self.series).lower() == \
+               IGNORED_CHARACTERS_RE.sub('', video.series).lower():
```

Comment:
Maybe worth adding a `sanitized_series` property on the `video` object what do you think?